webapp/WEB-INF/templates/admin/plugins/newsletter/send_newsletter.html

```diff
@@ -263,10 +263,10 @@
 		<tr>
 			<td style="padding: 0 20px; font-family: sans-serif; font-size: 12px; line-height: 15px; text-align: center; color: #ffffff;">
 				<#if unsubscribe?exists>
-		            <#if unsubscribe = "TRUE">
+		            <#if unsubscribe??>
```

Comment:
?? means "exists and is not empty", but it could be false >> add : 
<#if unsubscribe?? && unsubscribe >

Reply:
Yes, I have applied this correction